strongbox-security/strongbox-security-api/src/main/java/org/carlspring/strongbox/configuration/StrongboxDelegatingPasswordEncoder.java

```diff
@@ -0,0 +1,72 @@
+package org.carlspring.strongbox.configuration;
+
+import com.hazelcast.util.Base64;
+import javax.annotation.PostConstruct;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+/**
+ *  This class encodes given raw String or Base64 encoded string.
+ *  If given string is Base64 Encoded it will attempt to decode the password and finally delegate it to the password encoder.
+ *
+ * @author mbharti
+ * @date 07/10/20
+ */
+@Component
+public class StrongboxDelegatingPasswordEncoder implements PasswordEncoder {
```

Comment:
Cosmetic: Braces on new lines (here and everywhere across the pull request), please. :)

Reply:
@carlspring Sorry! will format according to our coding convention